src/EnergyPlus/StateManagement.cc

```diff
@@ -313,7 +313,7 @@ void EnergyPlus::clearAllStates()
     EMSManager::clear_state();
     EvaporativeCoolers::clear_state();
     EvaporativeFluidCoolers::clear_state();
-    ExteriorEnergyUse::clear_state();
+    //ExteriorEnergyUse::clear_state();
```

Comment:
So over time, the StateManagement file will become moot as well.  All the clear_states will go through the data manager.  This is going to be great.